source/_integrations/alexa.smart_home.markdown

```diff
@@ -217,6 +217,7 @@ Alexa can link your Amazon account to your Home Assistant account. Therefore Hom
   - Click the Smart Home skill you just created.
   - Click `Enable to use`.
   - A new window will open to direct you to your Home Assistant's login screen.
+    note: If you get a 403 Unauthorized at this stage, please verify if you have IP Ban activated.
```

Comment:
I'm not sure if this comment is clear or what the intention is?

Right now it reads like: You need to activate it?

Additionally, it doesn't matter if IP Ban is activated or not?

Reply:
Hi @frenck , you are right, statement not completely clear. The problem is that after some tests, activated IP ban could prevent the conclusion of the procedure. I will try to better explain it.